versions.tf

```diff
@@ -1,7 +1,10 @@
 terraform {
-  required_version = ">= 0.13.1"
-
   required_providers {
-    aws = ">= 3.67"
+    aws = {
+      version               = ">= 3.67"
+      configuration_aliases = [aws.kms]
```

Comment:
```suggestion
     configuration_aliases = [aws.us-east-1]
```

Reply:
Since we now move the provider out shall we rename it? its important the aliassed provider has us-east-1 set otherwise this will not work